examples/pipeline_wav2letter/ctc_decoders.py

```diff
@@ -0,0 +1,15 @@
+from torch import topk
+
+
+class GreedyDecoder:
```

Comment:
You could generalize this file to be called "decoders.py" and also fold in things such as compute_error_rates

Reply:
This class is stateless. Can it be a function?

Reply:
It could be functional corresponding to a transform, but really it's a step towards our beamsearch work